Content.Server/Traits/TraitSystem.Functions.cs

```diff
@@ -677,3 +677,26 @@ public override void OnPlayerSpawn(EntityUid uid,
         entityManager.Dirty(uid, melee);
     }
 }
+
+
+    // <summary>
+    // Adds a Tag to something
+    // </summary>
+    [UsedImplicitly]
```

Comment:
```suggestion
// <summary>
// Adds a Tag to something
// </summary>
[UsedImplicitly]
```
A simple explanation, these needed to be at the same indentation as the object beneath them.